test/testdata/pipelinerun-on-comment-annotation.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: "on-comment"
+  annotations:
+    pipelinesascode.tekton.dev/target-namespace: "\\ .TargetNamespace //"
+    pipelinesascode.tekton.dev/on-comment: "^/hello-world"
+spec:
+  pipelineSpec:
+    tasks:
+      - name: task
+        displayName: "The Task name is Task"
+        taskSpec:
+          steps:
+            - name: task
+              image: registry.access.redhat.com/ubi9/ubi-micro
+              command: ["/bin/echo", "{{ trigger_comment }}"]
```

Comment:
I just tried this sample example and i don't see any value for "{{ trigger_comment }}"
```
$ tkn pr logs -f another-pr-pac-demo-gdmds -n test-pac-pipelines
[noop-task : noop-task] 

[savita@fedora test-pac]$ 

```

Reply:
weird i tried it here and it works https://github.com/chmouel/scratchmyback/blob/gitops/.tekton/on-comment.yaml#L23

Reply:
yeah i tried it again and it works, https://github.com/openshift-pipelines/pipelines-as-code/pull/1579/files/7b5a0fabd39ed2f4fcedac754b87fcf90e01d546#diff-616beb623567ec63b23dde755a4a75be69e087c3fc7ddb36516160a6928bd36a maybe it's a quoting problem on your input?

![image](https://github.com/openshift-pipelines/pipelines-as-code/assets/98980/1683618e-2152-4e25-81a4-2abd57be9c3b)

Reply:
Yup i tried it today and it works for me 
It was mistake in my yaml :(

Reply:
which mistake in your yaml? this should be reported on the console now, is it something that is good yaml but tekton controller don't report as bad? (i don't think i can do much about it but i wonder)